apps/system/js/bluetooth_transfer.js

```diff
@@ -483,6 +482,7 @@ var BluetoothTransfer = {
       if (transferInfo.received) {
         // Received file can be opened only
         nData.titleL10n = 'transferFinished-receivedSuccessful-title';
+        nData.contentType = transferInfo.contentType;
```

Comment:
After reboot,  transferInfo is not available because it wont receive 'bluetooth-opp-transfer-complete' event. The only data it could count on is notifications message data